src/interface/swarm.hpp

```diff
@@ -143,7 +155,7 @@ class Swarm {
 
 using SP_Swarm = std::shared_ptr<Swarm>;
 using SwarmVector = std::vector<SP_Swarm>;
-using SwarmMap = std::map<std::string, SP_Swarm>;
+using SwarmMap = std::unordered_map<std::string, SP_Swarm>;
```

Comment:
You don't have to do this, but anytime you are not using a vector it is a good idea to leave a comment explaining why. At least that is what I was told.

Reply:
Noted, although this `map<string, Object>` pattern is regularly used in non-compute parts of parthenon so I'm gonna just leave this as is at least for now